README.rst

```diff
@@ -1,21 +1,26 @@
 hnn-core
 ========
 
-.. image:: https://circleci.com/gh/jonescompneurolab/hnn-core.svg?style=svg
-   :target: https://circleci.com/gh/jonescompneurolab/hnn-core
-   :alt: CircleCi
+|tests| |CircleCI| |Codecov| |PyPI| |voila-binder|
+
+|HNN-GUI|
+
+.. |tests| image:: https://github.com/jonescompneurolab/hnn-core/actions/workflows/unit_tests.yml/badge.svg?branch=master
+   :target: https://github.com/jonescompneurolab/hnn-core/actions/?query=branch:master+event:push
 
-.. image:: https://api.travis-ci.org/jonescompneurolab/hnn-core.svg?branch=master
-    :target: https://travis-ci.org/jonescompneurolab/hnn-core
-    :alt: Build Status
+.. |CircleCI| image:: https://circleci.com/gh/jonescompneurolab/hnn-core.svg?style=svg
+   :target: https://circleci.com/gh/jonescompneurolab/hnn-core
 
-.. image:: https://codecov.io/gh/jonescompneurolab/hnn-core/branch/master/graph/badge.svg
+.. |Codecov| image:: https://codecov.io/gh/jonescompneurolab/hnn-core/branch/master/graph/badge.svg
 	:target: https://codecov.io/gh/jonescompneurolab/hnn-core
-	:alt: Test coverage
 
-.. image:: https://user-images.githubusercontent.com/11160442/178095018-35d2619a-6a82-4e27-91c9-ff2796fab435.png
-   :target: https://user-images.githubusercontent.com/11160442/178095018-35d2619a-6a82-4e27-91c9-ff2796fab435.png
-   :alt: HNN-core GUI
+.. |PyPI| image:: https://img.shields.io/pypi/dm/hnn-core.svg?label=PyPI%20downloads
+	:target: https://pypi.org/project/hnn-core/
+
+.. |voila-binder| image:: https://mybinder.org/badge_logo.svg
```

Comment:
this won't work. You should have something like `/voila/render/hnn_widget.ipynb`. See:

https://voila.readthedocs.io/en/stable/deploy.html#deployment-on-binder

Reply:
Hmm, I've tried a bunch of things and can't get it to find the right path.

Reply:
Their docs aren't very clear.